contracts/scripts/app.ts

```diff
@@ -30,6 +30,26 @@ const grants = [
   },
 ];
 
+// Define grantRounds to create (addresses are random)
+const rounds = [
+  {
+    owner: '0x9B4d61BB9b8B6CB33a75B9846f7E2C8d67d137fa',
+    token: '0xDe30da39c46104798bB5aA3fe8B9e0e1F348163F', // GTC address
+    startTime: '1627374646',
+    endTime: '1630079135',
+    metaPtr: 'https://metadata-pointer.com',
+    minContribution: '100',
+  },
+  {
+    owner: '0x9B4d61BB9b8B6CB33a75B9846f7E2C8d67d137fa',
+    token: '0x6b175474e89094c44da98b954eedeac495271d0f', // DAI address
```

Comment:
Let's use the checksummed address here (the rest seem to already be checksummed)
```suggestion
    token: '0x6B175474E89094C44Da98b954EedeAC495271d0F', // DAI address
```

Reply:
Resolved by the rebase (but I did fix up `GRANT_REGISTRY_ADDRESS` in 9723b29)